### PR TITLE
[MIG][13.0] hr employee birth name

### PR DESCRIPTION
**Pull request by Trivedi-Vacha-SerpentCS**

### CODE REVIEW

File: hr_employee_birth_name/readme/USAGE.rst [hunk 1]
```diff
@@ -1,3 +1,3 @@
 .. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
```
Comment (alexey-pelykh) [inline hr_employee_birth_name/readme/USAGE.rst]: This content is not needed, it's added automatically. Thus just remove the file

**Review by alexey-pelykh — CHANGES_REQUESTED** [review]

File: hr_employee_birth_name/readme/INSTALL.rst [hunk 2]
```diff
@@ -0,0 +1,8 @@
+To install this module, you need to:
```
Comment (alexey-pelykh) [inline hr_employee_birth_name/readme/INSTALL.rst]: This instruction is common for all modules and has no specific value

File: hr_employee_birth_name/readme/DESCRIPTION.rst [hunk 3]
```diff
@@ -0,0 +1,3 @@
+This module adds the following to employees :
```
Comment (alexey-pelykh) [inline hr_employee_birth_name/readme/DESCRIPTION.rst:1]: Useless description, what this module does?

File: hr_employee_birth_name/README.rst [hunk 4]
```diff
@@ -0,0 +1,95 @@
+===================
```
Comment (alexey-pelykh) [inline hr_employee_birth_name/README.rst]: Either regenerate this file or remove it

**Review by alexey-pelykh — CHANGES_REQUESTED** [review]

**Review by jarroyomorales — APPROVED** [review]
Just fix the readme as @alexey-pelykh said and squash some Translation commits. But still LGTM, no major changes. Thank you!

File: hr_employee_birth_name/readme/DESCRIPTION.rst (hunk 3 above)
Comment (Trivedi-Vacha-SerpentCS) [reply hr_employee_birth_name/readme/DESCRIPTION.rst:1]: Enable a field to add the birth name of an employee.

Comment (alexey-pelykh) [reply hr_employee_birth_name/readme/DESCRIPTION.rst:1]: "This module adds feature to provide birth name of an employee"

File: hr_employee_birth_name/models/hr_employee.py [hunk 5]
```diff
@@ -0,0 +1,10 @@
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class HrEmployee(models.Model):
+    _inherit = 'hr.employee'
+
+    birth_name = fields.Char(string="Birth Name",
+        help="Birth Name of an employee.")
```
Comment (jarroyomorales) [inline hr_employee_birth_name/models/hr_employee.py]: And pep8 is failing here

**Review by lfreeke — APPROVED** [review]
Functional test

**Review by alexey-pelykh — APPROVED** [review]

File: hr_employee_birth_name/README.rst [hunk 6]
```diff
@@ -1,95 +0,0 @@
-===================
```
Comment (jarroyomorales) [inline hr_employee_birth_name/README.rst]: Why delete the README? Just re run it without the Install.rst

**Review by jarroyomorales — CHANGES_REQUESTED** [review]

File: hr_employee_birth_name/README.rst (hunk 6 above)
Comment (alexey-pelykh) [reply hr_employee_birth_name/README.rst]: README.rst will be regenerated by bot anyhow

Comment (jarroyomorales) [reply hr_employee_birth_name/README.rst]: Oh really? I didnt know that!

**Comment by jarroyomorales** [comment]
Same comment than https://github.com/OCA/hr/pull/689#pullrequestreview-295560703, dont remove copyrights

**Comment by Saran440** [comment]
@Trivedi-Vacha-SerpentCS please, using `pre-commit` for update this module.

**Pull request closed without merging**